docs/openapi/docs.go

```diff
@@ -0,0 +1,163 @@
+// Package openapi GENERATED BY SWAG; DO NOT EDIT
+// This file was generated by swaggo/swag
+package openapi
+
+import "github.com/swaggo/swag"
+
+const docTemplate = `{
+    "schemes": {{ marshal .Schemes }},
+    "swagger": "2.0",
+    "info": {
+        "description": "{{escape .Description}}",
+        "title": "{{.Title}}",
+        "termsOfService": "https://www.bytebase.com/terms",
+        "contact": {
+            "name": "API Support",
+            "url": "https://github.com/bytebase/bytebase/",
+            "email": "support@bytebase.com"
+        },
+        "license": {
+            "name": "MIT",
+            "url": "https://github.com/bytebase/bytebase/blob/main/LICENSE"
+        },
+        "version": "{{.Version}}"
+    },
+    "host": "{{.Host}}",
+    "basePath": "{{.BasePath}}",
+    "paths": {
+        "/sql/advise": {
+            "get": {
+                "description": "Parse and check the SQL statement by your schema review policy.",
+                "consumes": [
+                    "*/*"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
+                    "Schema Review"
+                ],
+                "summary": "Check the SQL statement.",
+                "parameters": [
```

Comment:
is the parameter case sensitive?

Reply:
They are. Updated the parameter description